src/components/Map/Map.jsx

```diff
@@ -1,44 +1,41 @@
 import React, { useEffect, useState } from 'react';
-import { MapContainer, TileLayer, Marker, Popup, useMap, ZoomControl } from 'react-leaflet';
+import { MapContainer, TileLayer, useMap, ZoomControl, Tooltip, Marker } from 'react-leaflet';
 import { GeoSearchControl, OpenStreetMapProvider } from 'leaflet-geosearch';
-import icons from 'leaflet-color-number-markers';
-import PopUpBox from '../PopUpBox/PopUpBox';
+import L from 'leaflet';
```

Comment:
Can we rename the "L" to something more descriptive of what we are importing? I believe you can just write "import Leaflet from 'leaflet'" if L is the default export. 
https://stackoverflow.com/questions/39282253/how-can-i-alias-a-default-import-in-javascript